docs: PCP-1327 fixed VMware section in the cluster section (#1362)
spectrocloud/librarium: content/docs/04-clusters/02-data-center/03-vmware.md

The following prerequisites must be met before deploying a Kubernetes clusters in VMware:

> 1. vSpehre version 7.0 or above. vSphere 6.7 is supported but not recommended.

Reviewer: ```suggestion
1. vSphere version 7.0 or above. vSphere 6.7 is supported but not recommended as it reached end of general support in 2022.
```

<br />

# VMware Privileges

The vSphere user account that is deploying Palette must have the following minimum vSphere privileges. The **Administrator** role provides super-user access to all vSphere objects. For users without the **Administrator** role, one or more custom roles can be created based on the tasks being performed by the user.

> Permissions and privilieges vary depending on the vSphere version you are using. 

Reviewer 2: ```suggestion
Permissions and privileges vary depending on the vSphere version you are using. 
```

> Select the tab that corresponds with your vSphere versions.

Reviewer 2: ```suggestion
Select the tab that corresponds with your vSphere version.
```

<br />

<InfoBox>
If the network is a Distributed Port Group under a vSphere Distributed Switch (VDS),  ReadOnly access to the VDS without “Propagate to children” needs to be provided.
</InfoBox>

<Tabs identifier="vm-privileges">

> <Tabs.TabPane tab="8.0.x" key="8.0.x" >

Reviewer: ```suggestion
<Tabs.TabPane tab="8.0" key="8.0" >
```

||* Guest operation alias query|
||* Guest operation modifications|
||* Guest operation program execution|
||* Guest operation queries|
||**Interaction**|
||* Console interaction|
||* Power off|
||* Power on|
||**Provisioning**|
||* Allow disk access|
||* Allow file access|
||* Allow read-only disk access|
||* Allow virtual machine download|
||* Allow virtual machine files upload|
||* Clone template|
||* Clone virtual machine|
||* Create template from virtual machine|
||* Customize guest|
||* Deploy template|
||* Mark as template|
||* Mark as virtual machine|
||* Modify customization specification|
||* Promote disks|
||* Read customization specifications|
||**Service configuration**|
||* Allow notifications|
||* Allow polling of global event notifications|
||* Manage service configurations|
||* Modify service configuration|
||* Query service configurations|
||* Read service configuration|
||**Snapshot management**|
||* Create snapshot|
||* Remove snapshot|
||* Rename snapshot|
||* Revert to snapshot|
||**vSphere Replication**|
||* Configure replication|
||* Manage replication|
||* Monitor replication|
|**vSAN**|Cluster|
||ShallowRekey|

</Tabs.TabPane>

> <Tabs.TabPane tab="7.0.x" key="7.0.x" >

Reviewer: ```suggestion
<Tabs.TabPane tab="7.0" key="7.0" >
```
